snippets/dart_2_snippets.md

```diff
@@ -0,0 +1,858 @@
+## Available Dart 2 Builder-Compatible Code Snippets
+
+Below is a list of all available Dart 2 builder-compatible over_react snippets and their<br>
+triggers for Webstorm/IntelliJ and VS Code. 
+
+| Trigger  | Content |
+| -------: | ------- |
+| `or_stateless`   | dart 2 stateless component skeleton |
```

Comment:
Could we remove the underscores? Most of the IntelliJ Live templates I've seen don't have delimiters and are pretty terse

<img width="1019" alt="screen shot 2018-12-17 at 2 33 22 pm" src="https://user-images.githubusercontent.com/1713967/50116944-e1097c80-0208-11e9-9a7b-b0c2b46929d7.png">